flink-connector-postgres-cdc/src/main/java/com/ververica/cdc/connectors/postgres/source/fetch/PostgresScanFetchTask.java

```diff
@@ -92,12 +94,15 @@ public void execute(Context context) throws Exception {
         PostgresSnapshotSplitReadTask snapshotSplitReadTask =
                 new PostgresSnapshotSplitReadTask(
                         ctx.getConnection(),
+                        (PostgresReplicationConnection) ctx.getReplicationConnection(),
                         ctx.getDbzConnectorConfig(),
                         ctx.getDatabaseSchema(),
                         ctx.getOffsetContext(),
                         ctx.getDispatcher(),
                         ctx.getSnapshotChangeEventSourceMetrics(),
-                        split);
+                        split,
+                        "a",
+                        "b");
```

Comment:
The slotName and pluginName will always be 'a' and 'b'. Do we really need these parameters?

Reply:
😭, I forgot update here